src/gt4py/next/program_processors/runners/dace_iterator/itir_to_tasklet.py

```diff
@@ -419,6 +579,40 @@ def builtin_cast(
     )
 
 
+def builtin_make_const_list(
+    transformer: "PythonTaskletCodegen", node: itir.Expr, node_args: list[itir.Expr]
+) -> list[ValueExpr]:
+    di = dace_debuginfo(node, transformer.context.body.debuginfo)
+    args = [transformer.visit(arg)[0] for arg in node_args]
+    assert all([isinstance(x, (SymbolExpr, ValueExpr)) for x in args])
+    args_dtype = [x.dtype for x in args]
+    assert len(set(args_dtype)) == 1
+    dtype = args_dtype[0]
+
+    var_name = unique_var_name()
+    transformer.context.body.add_array(var_name, (len(args),), dtype, transient=True)
+    var_node = transformer.context.state.add_access(var_name, debuginfo=di)
+
+    for i, arg in enumerate(args):
+        if isinstance(arg, SymbolExpr):
+            transformer.context.state.add_edge(
+                transformer.context.state.add_tasklet(
+                    f"get_arg{i}", {}, {"val"}, f"val = {arg.value}"
+                ),
+                "val",
+                var_node,
+                None,
+                dace.Memlet(data=var_name, subset=f"{i}"),
+            )
+        else:
+            assert arg.value.desc(transformer.context.body).shape == (1,)
+            transformer.context.state.add_nedge(
+                arg.value, var_node, dace.Memlet(data=var_name, subset=f"{i}", other_subset="0")
```

Comment:
Correct me if I am wrong but I think that the Memlet is wrong, shouldn't `subset` and `other_subset` be switched?
And data should be `arg`?

Reply:
Good catch, fixed.